AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -245,6 +267,8 @@ - (void)reloadDataWithAnimationOptions:(ASDataControllerAnimationOptions)animati
       NSMutableArray *updatedIndexPaths = [NSMutableArray array];
       [self _populateFromEntireDataSourceWithMutableNodes:updatedNodes mutableIndexPaths:updatedIndexPaths];
       
+      [self _layoutNodesWithMainThreadAffinity:updatedNodes atIndexPaths:updatedIndexPaths];
```

Comment:
I thought about this for a while, but can't see what requires it.  Wouldn't the insertion / reload of nodes trigger a -layoutSubviews pass where the cell layout code will run?  That should happen on the same runloop, so I think theoretically the main thread cells will be handled before the next transaction flush occurs — so I'm not confident why we need these.

Could you add a comment above these calls indicating their purpose?  This file is full of comments, but I think they are important since the cody is tricky.